src/main.cpp

```diff
@@ -775,7 +774,7 @@ MiningCPID GetNextProject(bool bForce)
         }
         }
 
-        msMiningErrors = (msPrimaryCPID == "INVESTOR") ? "" : _("All BOINC projects exhausted.");
+        msMiningErrors = (!IsResearcher(msPrimaryCPID)) ? "" : _("All BOINC projects exhausted.");
```

Comment:
I think @tomasbrod preferred to check for positive. I agree, it is a little bit cleaner:

```
msMiningErrors = IsResearcher(msPrimaryCPID) ? _("All BOINC projects exhausted.") : "";